README.md

```diff
@@ -36,35 +36,47 @@ ga('set', 'anonymizeIp', true);
 ga('send', 'pageview');
 ```
 
-### DAP Account Snippet   
-  
-1. ...  
-2. ...  
-3. ...  
+## Digital Analytics Program Snippet
+For more information about DAP visit thier [site](https://www.digitalgov.gov/services/dap/). Snippet information taken from [DAP Github page](https://github.com/digital-analytics-program/gov-wide-code).
 
-Here is the model snippet:  
+#### Digital Analytics Program government-wide code
+Provides a JavaScript file for US federal agencies to link or embed in their websites to participate in the Digital Analytics Program.
 
-````
-...
-````
+The most current version of DAP GA code is:
 
+* [`Universal-Federated-Analytics.js`](Universal-Federated-Analytics.js) (full)
+* [`Universal-Federated-Analytics-Min.js`](Universal-Federated-Analytics-Min.js) (minified)
 
-## api.data.gov 
+#### Central hosting
 
-Each project that builds an API should ensure that it is tracking how the API is being used.  The easiest means for 18F to do this is by integrating its own [api.data.gov service](https://api.data.gov/).  
+The central URLs for the Digital Analytics Program JS snippet are:
+
+* `https://dap.digitalgov.gov/Universal-Federated-Analytics.js` (full)
+* `https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js` (minified)
+
+Agencies are encouraged to use the following HTML snippet to participate in the Digital Analytics Program:
+
+```html
+<!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
+<script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js"></script>
```

Comment:
I have a PR open for the DAP -- https://github.com/digital-analytics-program/gov-wide-code/pull/27 -- which adds the `async` attribute to the `<script>` tag. GA uses `async` for their own snippet already.